frontend/src/components/FileBrowser.js

```diff
@@ -376,6 +376,7 @@ const rowCenter = {
     display: 'flex',
     flexDirection: 'row',
     alignItems: 'center',
+    cursor: 'pointer',
```

Comment:
Before clicking on the fileBrowserLite, the cursor doesn't change to pointer which indicates 'can open'